### PR TITLE
addition of Danish v2 0

### PR DESCRIPTION
**Pull request by bilfri**
Hi Contributor Covenant team,
Here's my attempt at danish localisation of the code of conduct version 2.0
I'll be happy to discuss it with someone else and get a polished version out soon

### CODE REVIEW

**Comment by CoralineAda** [comment]
Thanks so much! I'll find some other native speakers to help review.

File: content/version/2/0/code_of_conduct.da.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Aftale om regler for deltagelse i fælleskabet
+
+## Vores løfte
+
+Som medlemmer og ledere lover vi at deltagelse i vores fællesskab bliver en chikanefri oplevelse for alle, uanset alder, kropsform, synlige eller usynlige handicap, etnicitet, køn, erfaringsniveau, uddannelse, social- eller økonomisk status, nationalitet, udseende, hudfarve, kaste, religion, eller seksuel orientering.
```
Comment (abjerner) [inline content/version/2/0/code_of_conduct.da.md]:
```suggestion
Som medlemmer, bidragsydere og ledere lover vi at deltagelse i vores fællesskab bliver en chikanefri oplevelse for alle, uanset alder, kropsform, synlige eller usynlige handicap, etnicitet, køn, erfaringsniveau, uddannelse, social- eller økonomisk status, nationalitet, udseende, hudfarve, kaste, religion, eller seksuel orientering.
```

File: content/version/2/0/code_of_conduct.da.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Vi lover at deltage og samarbejde på en måde, der bidrager til at gøre fællesskabet åbent, imødekommende, mangfoldigt, inkluderende og velfungerende.
+
+## Vores standarder
+
+Eksempler på hvordan der skabes en god stemning i vores fællesskab inkluderer:
+
+* At vise medfølelse og være venlig overfor andre.
+* At have respekt for andre meninger, synspunkter, og erfaringer.
+* At give og kærligt at modtage konstruktiv feedback.
```
Comment (abjerner) [inline content/version/2/0/code_of_conduct.da.md]:
```suggestion
* At give og kærligt modtage konstruktiv feedback.
```

File: content/version/2/0/code_of_conduct.da.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+* Brug af vulgært sprog eller vulgære billeder og seksuel opmærksomhed eller tilnærmelser af nogen art.
+* Ondsindet provokation, fornærmende eller nedladende kommentater, personlige eller politiske angreb.
+* Offentlig eller privat chikane.
+* Offentliggørelse af andres private information, f.eks. deling af deres e-mail eller postadresse uden deres tilladelse.
+* Anden opførsel som ikke hører til i et professionelt miljø.
+
+## Ansvar for håndhævelse
+
+Fællesskabets ledere er ansvarlige for at informere om og håndhæve vores standarder for acceptabel opførsel. Det sker ved på en retfærdig og proportionel måde at irettesætte opførsel som de bedømmer som uacceptabel, truende, offensiv eller skadelig.
+
```
Comment (abjerner) [inline content/version/2/0/code_of_conduct.da.md:30]: > Community leaders have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, and will communicate reasons for moderation decisions when appropriate.

This part seems to be left out when comparing to the English version. Maybe it should be added as well?

File: content/version/2/0/code_of_conduct.da.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+### 2. Advarsel
+
+**Alvorsgrad**: En overstrædelse gennem en enkel hændelse eller en række af handlinger.
+
+**Konsekvens**: En advarsel om konsekvenser hvis opførslen fortsætter. Desuden må vedkommende ikke interagere med de personer der er ramt af overtrædelsen, samt dem der er ansvarlige for håndhævelse af fællesskabsreglerne, i en angivet tidsperiode. Det inkluderer både intern kommunikation og ekstern kommunikation som f.eks. sociale medier. Overtrædelse af denne betingelse kan føre til midlertidig eller permanent udelukkelse fra fællesskabet.
+
+### 3. Midlertidig udelukkelse
+
+**Alvorsgrad**: En alvorlig overtrædelse af fælleskabets standarder, f.eks. genoptrædende episoder med uacceptabel opførsel.
```
Comment (abjerner) [inline content/version/2/0/code_of_conduct.da.md]:
```suggestion
**Alvorsgrad**: En alvorlig overtrædelse af fælleskabets standarder, f.eks. gentagende episoder med uacceptabel opførsel.
```

*gentagende* may be a better word than *genoptrædende*

**Review by abjerner — CHANGES_REQUESTED** [review]
Hi @bilfri and @CoralineAda 

I was pointed in direction of this PR as I'm a native Danish speaker. I think the suggested version does a good job of preserving the meaning from English version 👍 

I've submitted a review with request for a few changes, but really, those are minor details, and the PR should be fine to merge with or without them.

File: content/version/2/0/code_of_conduct.da.md (hunk 3 above)
Comment (bilfri) [reply content/version/2/0/code_of_conduct.da.md:30]: Ooooh, yes, this one is important! Should I make a new pull request, or can I make a review myself?

Comment (abjerner) [reply content/version/2/0/code_of_conduct.da.md:30]: @bilfri if you update the file in the `danish-v2_0` branch in your own fork, the changes should appear here:

https://github.com/bilfri/contributor_covenant/blob/danish-v2_0/content/version/2/0/code_of_conduct.da.md

You can either edit the file online through the GitHub website, or pull your branch down, add the missing section, and then push it back up.

Comment (bilfri) [reply content/version/2/0/code_of_conduct.da.md:30]: thx, I think it worked :) didn't know that changes committed after the pull request would be included - cool!

**Review by abjerner — APPROVED** [review]
Looks great now 👍

**Review by CoralineAda — APPROVED** [review]

**Review by CoralineAda — APPROVED** [review]

**Pull request merged**